src/pages/membership/components/approveYouthProfile/ApproveYouthProfile.tsx

```diff
@@ -31,19 +32,26 @@ type Params = {
 
 function ApproveYouthProfile(props: Props) {
   const [approvalSuccessful, setApprovalSuccessful] = useState(false);
+  const [showNotification, setShowNotification] = useState(false);
+
   const params = useParams<Params>();
   const { t } = useTranslation();
   const { data, loading: queryLoading } = useQuery<YouthProfileByApprovalToken>(
     PROFILE_BY_TOKEN,
     {
       variables: { token: params.token },
+      onError: () => toggleErrorNotification(),
     }
   );
   const [approveProfile, { loading }] = useMutation<
     ApproveYourProfileData,
     ApproveYouthProfileVariables
   >(APPROVE_PROFILE);
 
+  const toggleErrorNotification = () => {
```

Comment:
This isn't really an toggle since it always sets `showNotification` to true. I would remove this function and just call `setShowNotification(true)` instead.